radix-engine/src/errors.rs

```diff
@@ -286,6 +286,8 @@ pub enum SystemError {
     BlsError(String),
     InputDataEmpty,
 
+    CannotLockFeeInChildSubintent(usize),
```

Comment:
I think that this is a breaking change to the `RuntimeError` type which would cause issues similar to what we had with the node at the start of bottlenose. I think that we'd want to add this variant at the end of the `SystemError` enum.